### DISCUSSION TITLE
Welcome to Xrm-Templating-Language Discussions!

### DISCUSSION THREAD

**Discussion by DigitalFlow** [DigitalFlow/Xrm-Templating-Language#43] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fvakiliev** [comment]
Hello!

How to sum multiple numeric values? Any mathematical function?

For example (doesn't work):

> ${{
>     If(IsNull(Value("mobilephone")), 0, 20) +
>     If(IsNull(Value("emailaddress1")), 0, 10) +
>     If(IsNull(Value("and_telegram")), 0, 10) +
>     If(IsNull(Value("and_facebook")), 0, 10) +
>     If(IsNull(Value("and_linkedin")), 0, 10) +
>     If(IsNull(Value("and_vk")), 0, 10) +
>     If(IsNull(Value("and_instagram")), 0, 10)
> }}
>
